gradle/dependencies.gradle

```diff
@@ -18,12 +18,12 @@ ext {
   println("Navigation Native version: " + mapboxNavigatorVersion)
 
   version = [
-      mapboxMapSdk              : '10.7.0-beta.1',
+      mapboxMapSdk              : '10.7.0-rc.1',
       mapboxSdkServices         : '6.6.0',
       mapboxEvents              : '8.1.5',
       mapboxCore                : '5.0.2',
       mapboxNavigator           : "${mapboxNavigatorVersion}",
-      mapboxCommonNative        : '22.1.0-beta.1',
+      mapboxCommonNative        : '22.1.0-rc.1',
```

Comment:
We now have a mismatch between Common SDK pre-release versions coming in from Maps SDK and Nav Native. Since pre-releases don't come with binary compatibility guarantees, we should rather wait for Nav Native to bump the version as well. We need to extend [the verification script](https://github.com/mapbox/mapbox-navigation-android/blob/main/gradle/verify-common-sdk-version.gradle) to catch this case.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/6063

Reply:
Fixed using NN `109.0.0`. I've just pushed 👍